src/operator/tensor/elemwise_binary_op-inl.h

```diff
@@ -292,9 +292,7 @@ void ElemwiseBinaryOp::CsrCsrOp(mshadow::Stream<cpu> *s,
 
   OpBase::FillDense<IType>(s, next.shape_.Size(), IType(-1), req, next.dptr_);
   OpBase::FillDense<DType>(s, lhs_row.shape_.Size(), DType(0),  req, lhs_row.dptr_);
-  if (!same_lhs_rhs) {
-    OpBase::FillDense<DType>(s, rhs_row.shape_.Size(), DType(0), req, rhs_row.dptr_);
-  }
+  OpBase::FillDense<DType>(s, rhs_row.shape_.Size(), DType(0), req, rhs_row.dptr_);
```

Comment:
if its the same_lhs_rhs can we point to the same tensor for both lhs and rhs instead of calling filldense for rhs ?

Reply:
Thanks for the suggestion, changed